lib/cinch/plugins/karma.rb

```diff
@@ -43,7 +45,7 @@ def execute(m, item)
         item.downcase!
         init_karma(channel, item)
 
-        m.reply "Karma for #{item} is #{@storage.data[channel][item]}"
+        m.reply "#{item} has #{@storage.data[channel][item]} point(s) of karma"
```

Comment:
This is fine, but if you want to change the output of the bot, you need to fix the tests. 😂